### DISCUSSION TITLE
but how should I specific a dir ?

### DISCUSSION THREAD

**Discussion by yw662** [rizowski/eslint-watch#186] [Q&A] [answered]
Or just, what is the equivalent to `npx eslint ./src` ?
`npx esw ./src` says "No files matching the pattern "./src" were found."

**Comment by rizowski** [comment, marked as answer]
What file types are you trying to lint?

**Reply by yw662** [reply]
.ts and .tsx. I worked it out as `esw -w -c ./.eslintrc.json --ext .ts,.tsx './src/'`
But eslint never asked that.
